### DISCUSSION TITLE
proxy

### DISCUSSION THREAD

**Discussion by guiworldtv2** [streamlink/streamlink#4657] [Q&A] [answered]
hey guys, just good?
i've been trying here many times to use streamlink with proxy, does proxy still work here?
if yes how do you work with it?
I tried everything I can, thanks

**Comment by bastimeyer** [comment, marked as answer]
https://streamlink.github.io/cli/proxy.html

**Reply by guiworldtv2** [reply]
> https://streamlink.github.io/cli/proxy.html

I've already tested what's there, I don't know if my proxy doesn't work or if the commands there are invalid

**Reply by bastimeyer** [reply]
Then post the **entire command** and the **entire debug log output** ([`--loglevel=debug`](https://streamlink.github.io/cli.html#cmdoption-loglevel)). So far, all you've said is "I've tried something and it didn't work", which isn't particularly useful if you're asking for help.

**Reply by guiworldtv2** [reply]
streamlink --http-proxy http://47.74.9.142:80 https://www.youtube.com/watch?v=qsOj78u6rT4&ab_channel=RECORDTVJAPAN/live best

error: Unable to open URL: https://www.youtube.com/watch?v=qsOj78u6rT4 (HTTPSConnectionPool(host='www.youtube.com', port=443): Max retries exceeded with url: /watch?v=qsOj78u6rT4 (Caused by ProxyError('Cannot connect to proxy.', OSError('Tunnel connection failed: 400 Bad Request'))))

streamlink --url --default-stream  --output --stream-url --http-proxy https://200.12.133.6:8080 https://mitelefe.com/vivo/ best

[cli][info] Found matching plugin telefe for URL https://mitelefe.com/vivo/
error: Unable to open URL: https://mitelefe.com/vivo/ (HTTPSConnectionPool(host='mitelefe.com', port=443): Max retries exceeded with url: /vivo/ (Caused by ProxyError('Cannot connect to proxy.', OSError('Tunnel connection failed: 407 Proxy Authentication Required'))))
EOL

**Reply by bastimeyer** [reply]
> 1. Tunnel connection failed: 400 Bad Request
> 2. Tunnel connection failed: 407 Proxy Authentication Required
